data/tutorials/language/0it_00_values_functions.md

```diff
@@ -198,6 +177,17 @@ ha ha
 - : unit = ()
 ```
 
+<!-- user-defined single constructor variant example -->
+<!-- FIXME: create an example nested pattern matching -->
+This also works with user-defined types.
+```ocaml
+# type live_person = int * name;;
+type live_person = int * name
+
+# let age ((years, _) : live_person) = years
+val age : live_person -> int = <fun>
+```
```

Comment:
For these I get: 

```
# type live_person = int * name;;
Error: Unbound value live_person

# let age ((years, _) : live_person) = years;;
Error: Unbound type constructor live_person
```

Also, `;;` missing at the end of `let age...`

Reply:
Is the `name` type in scope? It's defined above.

Reply:
Ah! I bet that's it!

Reply:
That was it! Thank you!